pixz/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Maxim Moseychuk ("fshp") <franchesko.salias.hudro.pedros@gmail.com>
+
+pkgname=pixz
+pkgver=1.0.7
+pkgrel=2
+pkgdesc="Parallel, indexed xz compressor"
+arch=('i686' 'x86_64')
+url="https://github.com/vasi/pixz"
+license=('custom')
```

Comment:
https://github.com/vasi/pixz/blob/master/LICENSE
seems to indicate that it is BSD rather than custom

Reply:
I changed. But I keep the license file
https://wiki.archlinux.org/index.php/PKGBUILD#license